### PR TITLE
feature: add possibility to have a dedicated skipper nodepool

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -184,3 +184,11 @@ spec:
             name: skipper-default-filters
             optional: true
 {{ end }}
+{{ if eq .ConfigItems.enable_dedicate_nodepool_skipper "true"}}
```
Comment (mikkeloscar) [inline cluster/manifests/skipper/deployment.yaml:187]: The config item should be defined as `enable_dedicate_nodepool_skipper: "false"` in the default-configs file.

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml:187]: fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**